src/context/schemas/contact.schema.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/contact.schema.json",
+  "type": "object",
+  "allOf": [{ "$ref": "context.schema.json#/" }],
+  "properties": {
+    "type": { "const": "fdc3.contact" },
+    "id": {
+      "type": "object",
+      "properties": {
+        "email": { "type": "string" },
+        "FDS_ID": { "type": "string" }
```

Comment:
FDS_ID is something I'd associate with a 'person' object, rather than a contact object.  A 
'contact' is someone in my CMS, email, chat, etc.  A 'person' is a reference data object.

Reply:
Should I remove it again @nkolba (given that it is part of the FO definition)?

Reply:
I’d be in favor of removing then working with FO to clarify on their side

Reply:
I don't think we should introduce a new "person" object and thus create a separation between that and "contact". We are referencing an individual and there is only one of you and not different parts. It's not like "instrument" and "organization" where you need to reference specific parts for eg trading. I would of course be in favor to keep it in and line up with FO.

Reply:
@donbasuno -I'm fine with leaving as is.